pywt/_multilevel.py

```diff
@@ -1008,7 +1008,9 @@ def ravel_coeffs(coeffs, axes=None):
     ----------
     coeffs : array-like
         A list of multilevel wavelet coefficients as returned by
-        `wavedec`, `wavedec2` or `wavedecn`.
+        `wavedec`, `wavedec2` or `wavedecn`. This function is also compatible
+        with the output of `swt`, `swt2` and `swtn` if those functions were
+        called with `trim_approx=True`.
```

Comment:
nitpick: double backticks for typesetting code

Reply:
Agreed. I added a commit that address it more comprehensively across the whole code base. This may cause me to need to rebase some other open PRs if it is merged, but I think it is still worth doing.